ultrasonic/src/main/java/org/moire/ultrasonic/util/Util.java

```diff
@@ -153,7 +153,25 @@ public static boolean isLockScreenEnabled(Context context)
 	public static String getTheme(Context context)
 	{
 		SharedPreferences preferences = getPreferences(context);
-		return preferences.getString(Constants.PREFERENCES_KEY_THEME, "dark");
+		return preferences.getString(Constants.PREFERENCES_KEY_THEME, Constants.PREFERENCES_KEY_THEME_DARK);
+	}
+
+	public static void applyTheme(Context context)
+	{
+		String theme = Util.getTheme(context);
+
+		if (Constants.PREFERENCES_KEY_THEME_DARK.equalsIgnoreCase(theme) || "fullscreen".equalsIgnoreCase(theme))
```

Comment:
Btw I do not know what this "fullscreen" check is inside that function

Reply:
Seems legacy code. There is a lot of legacy code in this app... :shrug: